spec/views/groups/questions/written/answered/index.html.haml_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'groups/questions/written/answered/index', vcr: true do
```

Comment:
You need to test all of your view logic branches - what happens with no date, no question, etc.